client/src/components/UserSignUpForm.js

```diff
@@ -106,9 +101,10 @@ export class UserSignUpForm extends Component {
                 <Button onClick={this.clickLogin}>Login</Button>
               </div>
             ) : (
-              <Form onSubmit={this.handleSubmit}>
-                <Form.Field>
+              <Form size="large" onSubmit={this.handleSubmit}>
```

Comment:
Can you remove the `size="large" ` to keep the same size for the text as before?

Reply:
it is done